libs/packages/sam-formly/src/lib/formly/types/fileinfo.ts

```diff
@@ -27,10 +28,10 @@ import { FieldType } from '@ngx-formly/core';
           />
           <label [for]="id + '_' + i">
             <div class="sds-card__header sds-card__header--center">
-              <h3 class="sds-card__title ">{{ option.label }}</h3>
+            <usa-icon [icon]="'filetype-' + option.value.toString().toLowerCase()" [size]="'2x'"></usa-icon>
```

Comment:
This will not work with ZIP icon. The code for it is `file-zip`.

Reply:
Also on that note, ZIP icon is not populating, as a result of the above comment.